unittests/test_web.py

```diff
@@ -96,7 +96,7 @@ def test_organisation_publication_page(
     assert response.status_code == 200
 
     assert (
-        '<p class="lead">It looks like you publish quarterly with a time lag of more than a quarter, so the maximum you can score for IATI data is 85.0 points. The total points for the relevant indicators have been adjusted accordingly.'
+        '<p class="lead">It looks like you publish quarterly with a time lag of more than a quarter, so the maximum you can score for IATI data is 84.99 points. The total points for the relevant indicators have been adjusted accordingly.'
```

Comment:
And if the `33.3` should be `33.33`, and same for `66.7` figure, then the max score should be `85.00`, I think, even when rounding to do digits:

```
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = 100 / 1.5
>>> a
66.66666666666667
>>> b = a * 0.775
>>> b
51.66666666666667
>>> c = b + 33.33
>>> c
84.99666666666667
>>> round(c, 2)
85.0
>>> 
```

Unless the weighting figures have been updated?

Reply:
Yes, that's right, I've updated it to fix it.